packages/app/src/components/nomination/nominationShow.js

```diff
@@ -0,0 +1,43 @@
+import React, { useEffect, useState } from 'react';
+import NominationBanner from './nominationBanner'
+import nominationsAPI from '../../utils/API/nominationsAPI';
+
+const NominationShow = ({ match: { params: { id } } }) => {
+  const [NominationData, setNominationData] = useState({})
+  const [errorMessage, setErrorMessage] = useState(null)
+
+  useEffect(() => {
+    nominationsAPI.fetchNomination(id)
+      .then(function (response) {
+        const nomination = response.data.nomination
+        setNominationData(nomination)
+     })
```

Comment:
Nitpick: It looks like the indentation is off here, and for the following three lines as well.